charts/substra-backend/templates/deployment-server.yaml

```diff
@@ -222,33 +222,13 @@ spec:
           - name: statics
             mountPath: /usr/src/app/backend/statics
       {{- include "common.waitPostgresqlInitContainer" . | nindent 6 }}
-      - name: init-migrate
-        image: {{ include "substra-backend.images.name" (dict "img" .Values.server.image "defaultTag" $.Chart.AppVersion) }}
-        command: ['python', 'manage.py', 'migrate']
-        securityContext:
-          allowPrivilegeEscalation: false
-          runAsNonRoot: true
-          seccompProfile:
-            type: RuntimeDefault
-          capabilities:
-            drop: ["ALL"]
-        envFrom:
-          - configMapRef:
-              name: {{ include "substra.fullname" . }}-orchestrator
-          - secretRef:
-              name: {{ include "substra.fullname" . }}-objectstore
-          - configMapRef:
-              name: {{ include "substra.fullname" . }}-database
-          - configMapRef:
-              name: {{ include "substra.fullname" . }}-settings
-          - secretRef:
-              name: {{ include "substra-backend.database.secret-name" . }}
-        env:
-        - name: DJANGO_SETTINGS_MODULE
-          value: backend.settings.{{ .Values.settings }}
```

Comment:
I'm not sure I follow the rationale for removing the `init-migrate` container only in the `server` app. Or maybe that's still work in progress?

Reply:
At least the `*-events` pod. Actually it seems like it's injected using the helper template (see [here](https://github.com/Substra/substra-backend/blob/main/charts/substra-backend/templates/deployment-api-events.yaml#L111) for instance):
```yaml
        {{- include "common.waitInitMigrationsInitContainer" . | nindent 8}}
```

Reply:
I am not sure to understand your comment.`init-migrate` is different from `common.waitInitMigrationsInitContainer`, and only appears in the server. I cannot see `init-migrate` in `*-events` . `init-migrate` actually applies migration (but without user mgmt, therefore duplicating features in the job). `common.waitInitMigrationsInitContainer` waits for migration.  In this PR, we are removing `init-migrate` from the server and replacing it with `common.waitInitMigrationsInitContainer`.

Reply:
Sorry indeed I have been confused with the naming of containers: `init-migrate` != `wait-init-migrations`. Thanks for helping clarifying that.